src/plugins/unified_search/public/saved_query_management/saved_query_management_list.tsx

```diff
@@ -236,9 +224,9 @@ export const SavedQueryManagementList = ({
     const fetchIdValue = ++currentPageFetchId.current;
 
     try {
-      const trimmedSearchTerm = searchTerm.trim();
+      const preparedSearch = escapeKuery(searchTerm.trim()).split(/\s+/).join(' AND ');
       const { total, queries } = await savedQueryService.findSavedQueries(
-        trimmedSearchTerm ? `${trimmedSearchTerm}*` : undefined,
+        preparedSearch ? `*${preparedSearch}*` : undefined,
```

Comment:
I'm not exactly sure what the intention here is (and to be fair I'm not exactly sure what it was before either) - are we wanting every term inside the search to show up somewhere in the title? Do we want partial matches (e.g. if my title is "hello" should "ello" work, or if my title is "hello there" should "hello world" work)?

In general I think this section of code should be as "dumb" as possible (just send `searchTerm` directly to the client-side service).

Reply:
Here's a diff along the lines of what I think might make more sense, maybe we can sync about it:

[patch.txt](https://github.com/elastic/kibana/files/14103270/patch.txt)

Reply:
Also as we discussed, I've refined how the search functionality works, and I think it should be an improvement now. And since we have the dedicated `_is_duplicate_title` endpoint now, I've also moved all the KQL logic to the back end so the endpoint just takes a raw search term now.